pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -93,7 +93,11 @@ class ClassTypeHintMatch:
 _TEST_FIXTURES: dict[str, list[str] | str] = {
     "aioclient_mock": "AiohttpClientMocker",
     "aiohttp_client": "ClientSessionGenerator",
+    "area_registry": "AreaRegistry",
     "caplog": "pytest.LogCaptureFixture",
+    "device_registry": "DeviceRegistry",
+    "entity_registry": "EntityRegistry",
+    "issue_registry": "IssueRegistry",
```

Comment:
Can we keep these in alphabetical order?

Reply:
My alphabet brain had a mallfunction :)